kubeadm/scripts/Install-Containerd.ps1

```diff
@@ -77,21 +77,19 @@ $env:Path += ";$global:ConainterDPath"
 containerd.exe config default | Out-File "$global:ConainterDPath\config.toml" -Encoding ascii
 #config file fixups
 $config = Get-Content "$global:ConainterDPath\config.toml"
-$config = $config -replace "bin_dir = (.)*$", "bin_dir = `"c:/opt/cni/bin`""
-$config = $config -replace "conf_dir = (.)*$", "conf_dir = `"c:/etc/cni/net.d`""
+$config = $config -replace "bin_dir = (.)*$", "bin_dir = `"C:\\Program Files\\containerd\\cni\\bin`""
+$config = $config -replace "conf_dir = (.)*$", "conf_dir = `"C:\\Program Files\\containerd\\cni\\conf`""
 $config | Set-Content "$global:ConainterDPath\config.toml" -Force 
 
-mkdir -Force c:\opt\cni\bin | Out-Null
-mkdir -Force c:\etc\cni\net.d | Out-Null
-
 Write-Output "Getting SDN CNI binaries"
-DownloadFile "c:\opt\cni\cni-plugins.zip" https://github.com/microsoft/windows-container-networking/releases/download/v0.2.0/windows-container-networking-cni-amd64-v0.2.0.zip
-Expand-Archive -Path "c:\opt\cni\cni-plugins.zip" -DestinationPath "c:\opt\cni\bin" -Force
+mkdir -Force $global:ConainterDPath\cni | Out-Null
+DownloadFile "$global:ConainterDPath\cni\cni-plugins.zip" https://github.com/microsoft/windows-container-networking/releases/download/v0.2.0/windows-container-networking-cni-amd64-v0.2.0.zip
+Expand-Archive -Path "$global:ConainterDPath\cni\cni-plugins.zip" -DestinationPath "$global:ConainterDPath\cni\bin" -Force
 
 Write-Output "Registering ContainerD as a service"
 containerd.exe --register-service
 
 Write-Output "Starting ContainerD service"
 Start-Service containerd
 
-Write-Output "Done - please remember to add '--cri-socket `"npipe:////./pipe/containerd-containerd`"' to your kubeadm join command"
\ No newline at end of file
+Write-Output "Done - please remember to add '--cri-socket `"npipe:////./pipe/containerd-containerd`"' to your kubeadm join command"
```

Comment:
fyi - This shouldn't be needed 1.25+.  Kubeadm automatically detects this after 1.25. It's ok to leave a note though.